### DISCUSSION TITLE
Golang+Vue轻松构建Web应用 - Qinng

### DISCUSSION THREAD

**Discussion by giscus** [qingwave/qingwave.github.io#68] [Announcements]
# Golang+Vue轻松构建Web应用 - Qinng

最近疫情在家，空闲时间比较多，整理下之前写的Golang项目Weave，补充了一些功能，加了前端实现。作为一个Web应用模板，也算是功能比较齐全了，现将开发过程中遇到的一些问题、项目特性总结下。 介绍 W

https://qingwave.github.io/golang-vue-starter/

**Comment by yellowfliag1** [comment]
您好，按照您的步骤去运行了项目，但是在make go 的时候出现了如下错误：
[error] failed to initialize database, got error failed to connect to `host=localhost user=postgres database=weave`: server error (FATAL: database "weave" does not exist (SQLSTATE 3D000))
{"level":"fatal","msg":"Init server failed: failed to connect to `host=localhost user=postgres database=weave`: server error (FATAL: database \"weave\" does not exist (SQLSTATE 3D000))","time":"2022-05-04T00:48:31-07:00"}
exit status 1
make: *** [run] 错误 1
请问您有遇到这个错误吗，您是怎么解决的，谢谢！

**Reply by qingwave** [reply]
漏了一步，需要先登录到postgres中创建下数据库，等会我补下
1. `make exec-db`
2. `create database weave`

**Reply by yellowfliag1** [reply]
这个项目目前是只能管理本地主机的容器吗？如果想管理远程的docker您觉得难度怎么样

**Reply by qingwave** [reply]
目前是只创建本地容器。如果要上生产，建议直接上k8s容器编排软件，比较方便管理和扩展；或者使用docker swarm也可以

**Reply by yellowfliag1** [reply]
请问仪表盘展示的是什么数据？为什么我本地安装后仪表盘展示的数据和您博客展示的一样，是写死的数据吗？

**Comment by Hexiaopi** [comment]
细致的看了一下，非常棒，值得学习。大佬有推荐类似值得参考的项目吗？

**Reply by qingwave** [reply]
可以Github上找以下，类似go-admin的项目应该有不少
